### PR TITLE
Enable to Leave network card without ip address

### PR DESCRIPTION
**Pull request by MalloZup**
fix issue #502

Testacc OK and tested With Ubuntu domain we Don't bail out with error

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,20 +1,16 @@
-cloud.google.com/go v0.26.0 h1:e0WKqKTd5BnrG8aKH3J3h+QvEIQtSUcf2n5UZ5ZgLtQ=
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```
Comment (MalloZup) [inline go.sum]: Regarding go.sum it can make prs unclean. Maybe we could ignore the file

Comment (dmacvicar) [reply go.sum]: No. You should set your `-mod` to `vendor` or `readonly` manually or in `GOFLAGS` and not checkin go.sum changes unless you are updating  dependencies.

https://github.com/golang/go/wiki/Modules

Comment (MalloZup) [reply go.sum]: ok i will update it

Comment (MalloZup) [reply go.sum]: thx

Comment (MalloZup) [reply go.sum]: done

**Pull request merged**